src/containers/Sidebar/SidebarNavItems.js

```diff
@@ -4,7 +4,9 @@ import { connect } from 'react-redux';
 import { Nav } from 'reactstrap';
 import SidebarItem from './SidebarItem';
 
-const SidebarNavItems = ({ navConfig, sidebarItemsOrder }) => {
+import { AppSidebarMinimizer } from '@coreui/react';
```

Comment:
@titkov49 This component will let us reliant on `coreui` again and I suppose this is not intended. Can we build the minimizer component by ourselves?

Reply:
I suppose we can't because we use core-ui layout and Sidebar relies on body, Header and Footer. This would require to rewrite them as well in order to view everything correct. I don't think that it's really hard but still this is more complex task